scss/reset/_customProperties.scss

```diff
@@ -0,0 +1,5 @@
+@include responsiveVarContext--base() {
```

Comment:
1. Let's assign a custom property with a default value to avoid redundant properties
2. Would the below work as an alternative to the mixin? It may be a bit more readable and easier to work with this way:

```sass
//
// Default value
//
:root {
	--responsiveBase: $base;
}

//
// Breakpoint values
//
@each $breakpoint, $responsiveBaseValue in (
	medium: <MEDIUM_BASE_VALUE>,
	large: <LARGE_BASE_VALUE>,
	huge: <HUGE_BASE_VALUE>
) {
	@include atMediaUp($breakpoint) {
		:root {
			--responsiveBase: $responsiveBaseValue;
		}
	}
}
```

```sass
// with the above scss setting the
// custom property value, we can turn this...
.thingWithResponsiveBase {
	padding-left: $space;
	padding-right: $space;
	padding-left: var(--baseSpacing);
	padding-right: var(--baseSpacing);
}

// into this
.thingWithResponsiveBase {
	padding-left: var(--responsiveBase);
	padding-right: var(--responsiveBase);
}
```

Reply:
Ok, something like this could definitely work. I'll give it a try!